### PR TITLE
Fixing form regressions

### LINKED ISSUE #12589: Unable to use forms in a focus item
### Mautic Version

5.0.x series

### PHP version

8

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When I try to use data capture for the focus item and select a form, there is an error.

### How can we reproduce this issue?

Step 1: Create a focus item and choose collect data in the builder
Step 2: Select a form, check the logs for an error and notice that no form is displayed.

### Relevant log output

```shell
[2023-07-26 15:16:32] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "text.html.twig" (looked into: /var/www/html/vendor/knplabs/knp-menu/src/Knp/Menu/Resources/views, /var/www/html/templates, /var/www/html/vendor/symfony/twig-bridge/Resources/views/Form)." at /var/www/html/app/bundles/FormBundle/Resources/views/Builder/form.html.twig line 78 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"text.html.twig\" (looked into: /var/www/html/vendor/knplabs/knp-menu/src/Knp/Menu/Resources/views, /var/www/html/templates, /var/www/html/vendor/symfony/twig-bridge/Resources/views/Form). at /var/www/html/app/bundles/FormBundle/Resources/views/Builder/form.html.twig:78)"} {"hostname":"mautic-web","pid":2076}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

### LINKED ISSUE #12627: HTML rendered when creating a checkbox group on Mautic 5 forms
### Mautic Version

5.0.x series

### PHP version

8

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When I create a checkbox group with labels I see the following HTML on the field when I save it.

screenshot-8080-mautic-mautic-73cmi7ovu7v ws-eu102 gitpod io-2023 08 04-16_03_01

Note it is also rendered on the actual form:

screenshot-8080-mautic-mautic-73cmi7ovu7v ws-eu102 gitpod io-2023 08 04-16_10_19

### How can we reproduce this issue?

Step 1: Create a form
Step 2: Add a checkbox group with two options (in my case I also added field attribute classes)
Step 3: Save the field and check the HTML output on the form - this should not be showing, maybe a hangover from the twig refactoring.

### Relevant log output

```shell
N/A
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by escopecz**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/12589, maybe even fixes https://github.com/mautic/mautic/issues/12627

#### Description:

This is fixing regression bugs introduced during the Twig migration when testing other PRs.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Follow steps in https://github.com/mautic/mautic/issues/12589
3. And also https://github.com/mautic/mautic/issues/12627

### CODE REVIEW

File: app/bundles/FormBundle/Resources/views/Builder/form.html.twig [hunk 1]
```diff
@@ -66,29 +66,15 @@
...
-                  <!-- start: "{{ theme }}{{ template }}" -->
-                  {{- include(theme ~ f.type ~ '.html.twig', {
-                          'field': f.convertToArray,
-                          'id': f.alias,
-                          'formName': formName,
-                          'fieldPage': (pageCount - 1),
-                          'mappedFields': mappedFields,
-                          'inBuilder': inBuilder,
-                          'fields': fields,
+                  {{- include([theme ~ f.type ~ '.html.twig', template], {
```
Comment (escopecz) [inline app/bundles/FormBundle/Resources/views/Builder/form.html.twig:69]: This syntax allows Twig to try if the template exists in the theme and if not, use the default one. See https://twig.symfony.com/doc/1.x/tags/include.html#:~:text=You%20can%20also%20provide%20a%20list%20of%20templates%20that%20are%20checked%20for%20existence%20before%20inclusion.%20The%20first%20template%20that%20exists%20will%20be%20included

**Review by PatrickJenkner — APPROVED** [review]
Just tested it and it fixes both problems 👍

**Review by kuzmany — APPROVED** [review]
Good 👍

**Comment by mollux** [comment]
aha, this is a better approach then https://github.com/mautic/mautic/pull/12606, closing that one :)

**Pull request merged**